easybuild/easyconfigs/l/libyaml/libyaml-0.1.4-intel-2015a.eb

```diff
@@ -0,0 +1,28 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Nils Christian <nils.christian@uni.lu>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'libyaml'
+version = '0.1.4'
```

Comment:
the homepage says "The current release of LibYAML: 0.1.5 (2014-02-04)."

I can see there's also a 1.0.6 tarball, but maybe this is not an official release? If you can find an announcement somewhere for 0.1.6, sure, use that.